[2021-07-16 16:18:32,749] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:17:26.874235+00:00 [queued]>
[2021-07-16 16:18:32,820] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:17:26.874235+00:00 [queued]>
[2021-07-16 16:18:32,821] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:18:32,823] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-16 16:18:32,825] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:18:32,862] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-16T16:17:26.874235+00:00
[2021-07-16 16:18:32,889] {standard_task_runner.py:52} INFO - Started process 7214 to run task
[2021-07-16 16:18:32,902] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-16T16:17:26.874235+00:00', '--job-id', '1002', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmpai990e5c', '--error-file', '/tmp/tmp3naj5mmg']
[2021-07-16 16:18:32,904] {standard_task_runner.py:77} INFO - Job 1002: Subtask restart_test
[2021-07-16 16:18:33,041] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:17:26.874235+00:00 [running]> on host 13346afafa5b
[2021-07-16 16:18:33,226] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-16T16:17:26.874235+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-16T16:17:17.530343+00:00
[2021-07-16 16:18:39,046] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210716T161726, start_date=20210716T161832, end_date=20210716T161839
[2021-07-16 16:18:39,138] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-16 16:18:39,203] {local_task_job.py:151} INFO - Task exited with return code 0
